integration/installFPC.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
this script fails to me (even after fixing package.sh). Haven't debug yet what causes it ..

Reply:
I updated it. I think it had similar issues related to `$(PWD)` :)